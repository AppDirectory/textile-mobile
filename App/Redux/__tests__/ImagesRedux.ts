import actions, { reducer } from '../ImagesRedux'

const initialState = reducer(undefined, {} as any)

describe('images stories', () => {
  describe('initial state', () => {
    it('should match snapshot', () => {
      expect(initialState).toMatchSnapshot()
    })
  })
})