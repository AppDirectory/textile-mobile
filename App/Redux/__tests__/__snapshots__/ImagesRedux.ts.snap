// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`images stories initial state should match snapshot 1`] = `
Object {
  "images": Object {
    "asset://bar": Object {
      "uri": "asset://bar",
    },
    "asset://baz": Object {
      "uri": "asset://baz",
    },
    "asset://foo": Object {
      "uri": "asset://foo",
    },
  },
  "processing": false,
}
`;
